internal/pkg/cli/svc_deploy.go

```diff
@@ -622,41 +622,18 @@ func (o *deploySvcOpts) stackConfiguration() (cloudformation.StackConfiguration,
 	var conf cloudformation.StackConfiguration
 	switch t := mft.(type) {
 	case *manifest.LoadBalancedWebService:
-		if o.targetApp.Domain == "" && t.HasAliases() {
-			log.Errorf(aliasUsedWithoutDomainFriendlyText)
-			return nil, errors.New("alias specified when application is not associated with a domain")
+		var appVersionGetter versionGetter
+		appVersionGetter, err = o.newAppVersionGetter(o.appName)
+		if err != nil {
+			return nil, fmt.Errorf("new app describer for application %s: %w", o.appName, err)
+		}
```

Comment:
Can we not put this in `validateLBWSRuntime` as well? I think ideally the only diff between `deploy` and `package` should be `deploy` deploys directly through CFN while `package` generates a ready-to-deploy artifact (CFN template and parameter file).

Reply:
Do you mean not passing `appVersionGetter` as a param to `validateLBWSRuntime`?

Reply:
oh nvm sorry i think this looks good!